tests/integration/security/sds_citadel_flow/sds_citadel_flow_test.go

```diff
@@ -85,16 +83,8 @@ func TestSdsCitadelCaFlow(t *testing.T) {
 				},
 			}
 
-			// Apply the policy to the system namespace.
-			deployment := tmpl.EvaluateOrFail(t, file.AsStringOrFail(t, "testdata/global-mtls.yaml"),
-				map[string]string{
-					"Namespace": ns.Name(),
-				})
-			g.ApplyConfigOrFail(t, systemNS, deployment)
-			defer g.DeleteConfigOrFail(t, systemNS, deployment)
-
-			// Sleep 3 seconds for the policy to take effect.
-			time.Sleep(3 * time.Second)
+			// Sleep 10 seconds for the policy to take effect.
+			time.Sleep(10 * time.Second)
```

Comment:
any reason to increase the waiting time from 3 to 10 seconds?

Reply:
3 seconds may not be enough for a policy to take effect. 10 seconds gives more time for a policy to take effect.